tests/e2e/setup.go

```diff
@@ -14,16 +13,16 @@ import (
 // Any method implemented for this struct will be ran when suite.Run() is called.
 type CCVTestSuite struct {
 	suite.Suite
-	coordinator   *ibctesting.Coordinator
-	providerChain *ibctesting.TestChain
-	consumerChain *ibctesting.TestChain
+	// coordinator   *ibctesting.Coordinator
```

Comment:
Sorry, I have removed this file on a local branch already. These changes will cause merge conflicts for you when I push.